src/app/deep-read/setup.sh

```diff
@@ -42,7 +42,7 @@ DIRECTORY=".venv"
 if [ ! -d "$DIRECTORY" ]; then
 # Control will enter here if $DIRECTORY doesn't exist.
     echo "Setting up venv"
-    python3 -m venv ./venv
+    python -m venv ./venv
```

Comment:
This should be python3 @NAshwinKumar coz many *nix systems still use python2 as python.